Source/CharismaModule/Public/CharismaEvents.h

```diff
@@ -388,79 +388,31 @@ struct FCharismaMessageHistoryResponse
 };
 
 USTRUCT(BlueprintType)
-struct FCharismaSpeechRecognitionResultEventAWSResultAlternatives
-{
-	GENERATED_USTRUCT_BODY()
-
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	FString Transcript;
-
-	MSGPACK_DEFINE_MAP(Transcript);
-};
-
-USTRUCT(BlueprintType)
-struct FCharismaSpeechRecognitionResultEventAWSResult
-{
-	GENERATED_USTRUCT_BODY()
-
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	FString ResultId;
-
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	float StartTime;
-
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	float EndTime;
-
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	bool IsPartial;
-
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	TArray<FCharismaSpeechRecognitionResultEventAWSResultAlternatives> Alternatives;
-
-	MSGPACK_DEFINE_MAP(ResultId, StartTime, EndTime, IsPartial, Alternatives);
-};
-
-USTRUCT(BlueprintType)
-struct FCharismaSpeechRecognitionResultEventAWSTranscript
+struct FCharismaSpeechRecognitionResultEvent
 {
 	GENERATED_USTRUCT_BODY()
-
+		
 	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	TArray<FCharismaSpeechRecognitionResultEventAWSResult> Results;
-
-	MSGPACK_DEFINE_MAP(Results);
-};
-
-USTRUCT(BlueprintType)
-struct FCharismaSpeechRecognitionResultEventAWSTranscriptEvent
-{
-	GENERATED_USTRUCT_BODY()
+	FString Text;
+	;
 
 	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	FCharismaSpeechRecognitionResultEventAWSTranscript Transcript;
-
-	MSGPACK_DEFINE_MAP(Transcript);
-};
-
-USTRUCT(BlueprintType)
-struct FCharismaSpeechRecognitionResultEventAWS
-{
-	GENERATED_USTRUCT_BODY()
+	bool SpeechFinal;
 
 	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	FCharismaSpeechRecognitionResultEventAWSTranscriptEvent TranscriptEvent;
+	bool IsFinal;
 
-	MSGPACK_DEFINE_MAP(TranscriptEvent);
+	MSGPACK_DEFINE_MAP(MSGPACK_NVP("speechFinal", SpeechFinal),
+		MSGPACK_NVP("isFinal", IsFinal),
+		MSGPACK_NVP("text", Text));
 };
 
-USTRUCT(BlueprintType)
-struct FCharismaSpeechRecognitionResultEvent
+struct FSpeechRecognitionErrorResult
 {
-	GENERATED_USTRUCT_BODY()
+	std::string message;
+	std::string errorOccuredWhen;
 
-	UPROPERTY(BlueprintReadOnly, Category = "Charisma|Event")
-	FString Transcript;
+	MSGPACK_DEFINE_MAP(message, errorOccuredWhen);
```

Comment:
to be consistent with other events returned from the server, let's use the convention of mapping lower-case fields from the MsgPack result into upper-case fields in the `struct` (like above)